### DISCUSSION TITLE
instable plot in huge point

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#519] [Q&A] [answered]
Hi all, is there anyone got same experience when plot huge data point to leafmap, I am test use 1500point data and its show in the map but gradually when adding some more data, there are no map show.

thanks

**Comment by giswqs** [comment, marked as answer]
No. Ipyleaflet/folium is not suitable for visualizing large vector data. See https://github.com/jupyter-widgets/ipyleaflet/issues/1134

**Reply by rh-afk** [reply]
Hi @giswqs thanks for your answer. 
Unfortunately I try to plot as many as point can be plot to leafmap and compare to folium. Until 1600 points data it works both but more than that leafmap couldn't while folium can. Probably you can reproduce the same problem?
thanks
